pkg/util/net/http.go

```diff
@@ -67,6 +70,9 @@ func SetTransportDefaults(t *http.Transport) *http.Transport {
 	if t.TLSHandshakeTimeout == 0 {
 		t.TLSHandshakeTimeout = defaultTransport.TLSHandshakeTimeout
 	}
+	if err := http2.ConfigureTransport(t); err != nil {
```

Comment:
Sorry. Didn't see your last comment. According to the CL, would it be simpler to default `ExpectContinueTimeout` to 0: https://go-review.googlesource.com/#/c/22035/2/src/net/http/transport.go ?